rest-api/src/main/java/io/nobt/rest/NobtRestApi.java

```diff
@@ -136,8 +143,8 @@ private void registerCreateNobtRoute() {
             res.header("Location", req.url() + "/" + nobt.getId().getValue());
```

Comment:
This link is probably wrong too.

Reply:
We should not use `url` here but create another factory that uses host and scheme.